src/models2/LoadMoreModel.es6.js

```diff
@@ -0,0 +1,27 @@
+import RedditModel from './RedditModel';
+import Record from '../apiBase/Record';
+
+import { COMMENT_LOAD_MORE } from './thingTypes';
+
+const T = RedditModel.Types;
+
+export default class LoadMoreModel extends RedditModel {
+  static type = COMMENT_LOAD_MORE;
+
+  static PROPERTIES = {
+    parentId: T.string,
+    children: T.arrayOf(T.string),
+    count: T.number,
+    depth: T.number,
+  }
+
+  static API_ALIASES = {
+    parent_id: 'parentId',
+  }
+
```

Comment:
load more has an id/name from api which happens to actually be the id its first child. yep..